k8s-rule.yaml

```diff
@@ -0,0 +1,1034 @@
+- message: Deprecated/removed Kubernetes API version 'extensions/v1beta1' is used for 'Deployment'. Consider using 'apps/v1'.
```

Comment:
Could we maybe get these rules separated into rulesets by the api group? i.e. rules for apps, rules for networking, etc..

Reply:
will look after it. need some time